### PR TITLE
fix(curriculum): English editing

### PR DESCRIPTION
**Pull request by waldoluisribeiro**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ var SpaceShuttle = function(targetPlanet){
 var zeus = new SpaceShuttle('Jupiter');
 ```
 
-The class syntax simply replaces the constructor function creation:
+The <code>class</code> syntax simply replaces the constructor function:
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md]: @waldoluisribeiro I am thinking `creation` should be left in here.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md [hunk 2]
```diff
@@ -29,16 +29,16 @@ class SpaceShuttle {
...
 <li> UpperCamelCase should be used by convention for ES6 class names, as in <code>SpaceShuttle</code> used above.</li>
 <li> The constructor method is a special method for creating and initializing an object created with a class. You will learn more about it in the Object Oriented Programming section of the JavaScript Algorithms And Data Structures Certification.</li></ul>
 </section>
 
 ## Instructions
 <section id='instructions'>
-Use <code>class</code> keyword and write a proper constructor to create the <code>Vegetable</code> class.
-The <code>Vegetable</code> lets you create a vegetable object, with a property <code>name</code>, to be passed to constructor.
+Use the <code>class</code> keyword and write a constructor to create the <code>Vegetable</code> class.
+The <code>Vegetable</code> class allows you to create a vegetable object with a property <code>name</code>, which is passed to the constructor.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md]:
```suggestion
The <code>Vegetable</code> class allows you to create a vegetable object with a property <code>name</code>that gets passed to the constructor.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@waldoluisribeiro See my suggestions.

**Review by RandellDawson — APPROVED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ forumTopicId: 301212
 
 ## Description
 <section id='description'>
-ES6 provides a new syntax to help create objects, using the keyword <dfn>class</dfn>.
-This is to be noted, that the <code>class</code> syntax is just a syntax, and not a full-fledged class based implementation of object oriented paradigm, unlike in languages like Java, or Python, or Ruby etc.
-In ES5, we usually define a constructor function, and use the <code>new</code> keyword to instantiate an object.
+ES6 provides new syntax to create objects, using the keyword <dfn>class</dfn>.
```
Comment (Manish-Giri) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md]:
```suggestion
ES6 provides new syntax to create objects, using the <dfn>class</dfn> keyword.
```

Being consistent with other such `dfn`/`code` statements.

Also, IMO, `"a new syntax"` was fine here.

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by RandellDawson** [comment]
@Manish-Giri Please merge if you are satisfied with the current changes.

**Review by Manish-Giri — APPROVED** [review]
LGTM.

**Pull request merged**